### DISCUSSION TITLE
[Question]: best way of archiving older recordings

### DISCUSSION THREAD

**Discussion by GoldieLocks89** [blakeblackshear/frigate#9944] [Ask A Question] [answered]
### What is your question:

Long story short, I have a stuttering recording issue that I think is caused by maxing out the IO on the spinning disks in my server. What I'd like to do is have frigate write new recordings to a fast NVMe drive where they'll remain for a day or whatever, before being archived off to my spinning disk and kept for a week or two.

I can't see any information about getting frigate to do this all for me so I assume it's not a feature that frigate currently supports, but I've seen other posts ([example](https://github.com/blakeblackshear/frigate/discussions/4963)) about getting unraid to do the file moving for me. The issue I have with that though is I assume once unraid has moved the files, I'll no longer be able to view the events in the frigate ui, and definitely won't be able to play the recordings back. And the prospect of having to manually look through folders full of 12 second videos to find what I need doesn't fill me with much joy!

Are there any suggestions you have for me about the best way of achieving this? Any recommended approaches? 

If not, can you make an enhancement to support this kind of functionality?

Thanks!

**Comment by NickM-27** [comment, marked as answer]
> The issue I have with that though is I assume once unraid has moved the files, I'll no longer be able to view the events in the frigate ui

no, that is totally incorrect, the way unraid handles this it appears as one big volume in frigate. every feature in frigate is fully usable throughout its lifetime in unraid

**Reply by GoldieLocks89** [reply]
Yes, you're right, sorry - I'd completely misunderstood the setup in unraid and how easy that makes it.
Thanks for the fast reply!
